src/main/osd/osd.h

```diff
@@ -292,6 +295,7 @@ typedef struct osdConfig_s {
     char profile[OSD_PROFILE_COUNT][OSD_PROFILE_NAME_LENGTH + 1];
     uint16_t link_quality_alarm;
     int16_t rssi_dbm_alarm;
+    int16_t rsnr_alarm;
```

Comment:
Is it a dBm value too? In which case the variable would be rsnr_dbm_alarm

Reply:
@TonyBlit bump

Reply:
rsnr is in db, but I don't think it's worth dragging it everywhere. the reason rssi_dbm exists is to differentiate from rssi (percentage) which in this case makes sense